OloEngine/src/OloEngine/Asset/AssetImporter.cpp

```
@@ -44,41 +44,47 @@ namespace OloEngine
         return s_SerializersMutex;
     }
 
-    static std::once_flag s_InitFlag;
     static std::atomic<bool> s_IsShuttingDown{ false };
 
     void AssetImporter::Init()
```

Cppcheck (2.20.0)

[style] The function 'Init' is never used.

(unusedFunction)